db/migrate/20170530173726_add_filepath_to_submissions.rb

```diff
@@ -0,0 +1,5 @@
+class AddFilepathToSubmissions < ActiveRecord::Migration[5.1]
+  def change
+    add_column :submissions, :filepath, :string
+  end
+end
```

Comment:
Will do for now. Though nce the system is deployed somewhere, I doubt we will be able to continue doing that.